docs/settings/spaces-settings.asciidoc

```diff
@@ -15,8 +15,8 @@ roles when Security is enabled.
 [cols="2*<"]
 |===
 | `xpack.spaces.enabled`
-  | Set to `true` (default) to enable Spaces in {kib}.
-    This setting is deprecated. Starting in 8.0, it will not be possible to disable this plugin.
+  | deprecated:[7.11.0] Starting in 8.0, it will not be possible to disable this plugin.
```

Comment:
Same problem with the display of this tooltip.

Reply:
Ditto, I've reverted the changes in this file.